src/routes/routes.js

```diff
@@ -63,5 +64,8 @@ router.patch('/api/v1/profiles/:username/notifications/:subscribe', [verifyToken
 router.get('/api/v1/profiles/notifications/all', verifyToken, Notification.getNotification);
 router.patch('/api/v1/profiles/notifications/:id/read', verifyToken, Notification.readOneNotification);
 router.patch('/api/v1/profiles/notifications/read/all', verifyToken, Notification.readAllNotification);
+router.post('/api/v1/bookmarks/:slug', verifyToken, Bookmark.createBookmark);
```

Comment:
I think you should use **get** because you are not sending any information in the body.

Reply:
Thanks, @placideirandora on it

Reply:
done